src/seckrit.py

```diff
@@ -85,5 +96,5 @@ def load_manifest(path):
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Fetches secrets from Google Cloud Secret Manager according to a YAML manifest.")
-    parser.add_argument("manifest_file", metavar="MANIFEST_FILE", help="YAML manifest file specifying which secrets to fetch and how they should be treated.")
+    parser.add_argument("--manifest", metavar="MANIFEST_FILE", help="YAML manifest file specifying which secrets to fetch and how they should be treated.")
```

Comment:
Now that we support two methods of providing the manifest, the help documentation should say this.

I would add `required=False` to this argument and leave it as-is, then add a second paragraph to the description of the `parser` and say the user can choose between this and `SECKRIT_MANIFEST`.